### PR TITLE
[bitnami/thanos] Set custom topology key for podAntiAffinityPreset

### PR DESCRIPTION
**Pull request by alirezaDavid**
Signed-off-by: Alireza Davoodi <alizdavoodi@gmail.com>

### Description of the change

Set custom topology key for podAntiAffinityPreset, e.g. `failure-domain.beta.kubernetes.io/zone`

Related MR:
https://github.com/bitnami/charts/pull/13435#issue-1442344820

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by carrodher** [comment]
Thanks for the contribution. Please note the GH actions are failing due to the following error
```
ERROR: Missing metadata for key: query.podAntiAffinityPresetTopologyKey
```

That is because when a new parameter is added, you need to specify some metadata to render the table appearing in the README file. You can use any other parameter as example

File: bitnami/thanos/Chart.yaml [hunk 1]
```diff
@@ -28,4 +28,4 @@ name: thanos
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/thanos
   - https://thanos.io
-version: 11.6.1
+version: 11.6.2
```
Comment (migruiz4) [inline bitnami/thanos/Chart.yaml:31]: Version `11.6.2` has been already released, could you please bump the chart version again? https://github.com/bitnami/charts/blob/main/bitnami/thanos/Chart.yaml

**Review by migruiz4 — CHANGES_REQUESTED** [review]
Hi @alirezaDavid, thank you for your contribution! 
The changes look good to me, but I need to ask you to bump the chart version again since version 11.6.2 was already released.

**Review by migruiz4 — APPROVED** [review]
Thank you for your contribution!

**Pull request merged**